installer/PowerToysSetupCustomActions/PowerToysSetupCustomActions.vcxproj

```diff
@@ -43,6 +43,22 @@
     <LinkIncremental>false</LinkIncremental>
     <IncludePath>..\..\src\common\Telemetry;$(IncludePath)</IncludePath>
   </PropertyGroup>
+  <ItemDefinitionGroup>
+    <PreBuildEvent>
+      <Command>
+      call cmd /C "copy ""$(ProjectDir)DepsFilesLists.h"" ""$(ProjectDir)DepsFilesLists.h.bk"""
+      call cmd /C "copy ""$(ProjectDir)..\PowerToysSetup\Product.wxs"" ""$(ProjectDir)..\PowerToysSetup\Product.wxs.bk""""
+      call pwsh.exe -File parseRuntimes.ps1 -depsjsonpath "$(ProjectDir)..\..\$(Platform)\$(Configuration)\PowerToys.ColorPickerUI.deps.json" -depsfileslistspath "$(ProjectDir)DepsFilesLists.h" -productwxspath "$(ProjectDir)..\PowerToysSetup\Product.wxs"
```

Comment:
Also, trying to build it locally from Developer command prompt for VS, it doesn't recognize pwsh.exe, but it does powershell.exe.. Let's try replacing that, CI should be able to recognize powershell.exe as well, right?